languageServer/src/TextDocumentWrapper.ts

```diff
@@ -45,132 +33,11 @@ export class TextDocumentWrapper implements ITextDocumentWrapper {
     return parseDocument(this.getText())
```

Comment:
`getYamlDocument` is no longer used anywhere (seems like it has been moved into the `languageHelper`) it should be removed from this class and the interface.